framework/validators/UniqueValidator.php

```diff
@@ -58,6 +58,15 @@ class UniqueValidator extends Validator
      * is the [[\yii\db\Query|Query]] object that you can modify in the function.
      */
     public $filter;
+	/**
+     * @var string the user-defined error message used when `targetAttribute` is an array. It may contain the following placeholders which
+     * will be replaced accordingly by the validator:
+     *
```

Comment:
Makes sense, but just so you know I copy-pasted this from `public $message` in `\yii\validators\Validator` so you may want to fix it there (and probably many other validator classes) too.